libraries/AP_RangeFinder/AP_RangeFinder.cpp

```diff
@@ -658,6 +658,15 @@ AP_RangeFinder_Backend *RangeFinder::find_instance(enum Rotation orientation) co
     return nullptr;
 }
 
+uint16_t RangeFinder::snr(enum Rotation orientation) const
+{
+    AP_RangeFinder_Backend *backend = find_instance(orientation);
```

Comment:
Should there be a check for rangefinder type here and return 0 if not ulanding / radar? Or am I missing where RangeFinder_State snr is initialized to 0?

Reply:
Ah, I see it now. It's a pointer t o AP_RangeFinder_Backend which is an instance of uLanding. Nvm :)